pkg/plugins/manager/loader/loader.go

```diff
@@ -131,6 +131,9 @@ func (l *Loader) Load(ctx context.Context, src plugins.PluginSource) ([]*plugins
 			l.recordError(ctx, validatedPlugin, err)
 			continue
 		}
+		if initializedPlugin.ID == "tempo" && !l.cfg.Features.TempoAlertingEnabled {
```

Comment:
nit: you can use

```suggestion
		if initializedPlugin.ID == coreplugin.Tempo && !l.cfg.Features.TempoAlertingEnabled {
```

Reply:
Changed in b7f1d78f1984ec475e7fbccd406852d713be3f65 + moved to plugins bootstrap